lib/shelley/src/Cardano/Wallet/Shelley/Compatibility.hs

```diff
@@ -0,0 +1,443 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DuplicateRecordFields #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE NumericUnderscores #-}
+{-# LANGUAGE PatternSynonyms #-}
+{-# LANGUAGE TupleSections #-}
+{-# LANGUAGE TypeFamilies #-}
+
+-- |
+-- Copyright: © 2020 IOHK
+-- License: Apache-2.0
+--
+-- Conversion functions and static chain settings for Shelley.
+
+module Cardano.Wallet.Shelley.Compatibility
+    ( Shelley
+    , ShelleyBlock (..)
+    , NodeVersionData
+    , TPraosStandardCrypto
+
+      -- * Chain Parameters
+    , mainnetVersionData
+    , testnetVersionData
+
+    , mainnetBlockchainParameters
+
+      -- * Genesis
+    , emptyGenesis
+    , genesisTip
+
+      -- * Conversions
+    , toShelleyHash
+    , toEpochSize
+    , toGenTx
+    , toPoint
+    , toSlotNo
+
+    , fromBlockNo
+    , fromShelleyBlock
+    , fromShelleyHash
+    , fromPreviousHash
+    , fromChainHash
+    , fromGenesisData
+    , fromNetworkMagic
+    , fromProtocolMagicId
+    , fromSlotNo
+    , fromTip
+    , fromTxAux
+    , fromTxIn
+    , fromTxOut
+    , fromPParams
+    ) where
+
+import Prelude
+
+
+import Cardano.Binary
+    ( serialize' )
+import Cardano.Chain.Common
+    ( unsafeGetLovelace )
+import Cardano.Chain.UTxO
+    ( Tx (..), TxAux, TxIn (..), TxOut (..), taTx )
```

Comment:
I don't think any of these apply to `Shelley`. 
We should try to use things from https://github.com/input-output-hk/cardano-node/blob/master/cardano-api/src/Cardano/Api/Types.hs#L108-L114, or better, use `Cardano.Api` here to interface with cardano-wallet

Reply:
👍 I simply removed them for now.